projects/fuzzing-puzzles/project.yaml

```diff
@@ -0,0 +1,13 @@
+homepage: https://github.com/google/fuzzer-test-suite
+primary_contact: kcc@google.com
+auto_ccs:
+  - "kevinwkt@google.com"
+  - "kodewilliams@google.com"
+  - "metzman@google.com"
+  - "mmoroz@google.com"
+
+sanitizers:
+  - address
+
+fuzzing_engines:
+  - libfuzzer
```

Comment:
@jonathanmetzman do you think it can be interesting to create a separate project for AFL? I think it could make sense, if we decided to fork AFL and work on improving it...